### ISSUE TITLE
Other implementations WIP

### ISSUE THREAD

**Issue by cburgmer** [cburgmer/json-path-comparison#10]
Just to keep track of what's already half way implemented, but not merged yet:

- [ ] [PHP_remorhaz-jsonpath](https://github.com/cburgmer/json-path-comparison/pull/6)
  See PR, #6 
- [ ] [Golang_github.com-bhmj-jsonslice](https://github.com/cburgmer/json-path-comparison/tree/Golang_github.com-bhmj-jsonslice)
  Open question around correct API usage.
- [ ] [Golang_github.com-yalp-jsonpath](https://github.com/cburgmer/json-path-comparison/tree/Golang_github.com-yalp-jsonpath)
  Seems to drag down the consensus a lot due to many features missing
- [ ] [Perl_JSON-Path](https://github.com/cburgmer/json-path-comparison/tree/Perl_JSON-Path)
  Indeterministic output, #4
- [ ] [Bash_JSONPath.sh](https://github.com/cburgmer/json-path-comparison/tree/Bash_JSONPath.sh)
  Does not work cross platform (OS X has issues)

**Comment by cburgmer** [comment]
I would disqualify Golang_github.com-yalp-jsonpath as its README states it is not suggested for production use.

**Comment by remorhaz** [comment]
It would be a nice feature to ignore implementation while calculating consensus, but evaluate it's work later (maybe show results in a different table or mark it's colunn somehow)

**Comment by cburgmer** [comment]
What you are implying though is that we can identify libraries not worthy contributing to the consensus though, right? I'm not sure how we could objectively define such a criteria.

**Comment by remorhaz** [comment]
I agree.

**Comment by brettz9** [comment]
Might [JSONata](https://github.com/jsonata-js/jsonata) be added to the list? It looks to be quite robust and of interest for such comparisons...

**Comment by cburgmer** [comment]
I couldn't find good documentation on jsonata's query language - from a
first limited glimpse there seems to be not much intentional overlap with
JSONPath?
If indeed they are fundamentally different, a comparison across the syntax
deviations would be a much harder undertaking.
I haven't looked into the SQL's standard of JSONPath yet much (see branch
on Postgres), this sounds like a similar harder issue due to differences in
the syntax.

**Comment by brettz9** [comment]
Yeah, sorry, I had just taken a mental note that it had a very robust API, but I had forgotten it is quite different. I think these pages give some idea of the basics: http://docs.jsonata.org/simple and http://docs.jsonata.org/predicate . Very interesting re: SQL implementing its own JSONPath...
